packages/js/product-editor/src/components/variations-table/variation-stock-status-form/variation-stock-status-form.tsx

```diff
@@ -157,7 +157,11 @@ export function VariationStockStatusForm( {
 						label={ __( 'Available stock', 'woocommerce' ) }
 						help={ errors.stock_quantity }
 						value={ value.stock_quantity }
-						onChange={ handleStockQuantityInputControlChange }
+						onChange={ ( val ) => {
+							if ( val ) {
```

Comment:
Could this be handled inside of `handleStockQuantityInputControlChange` instead?